File: C/PRAK205-2210817110005-MIZZUDDINABDISSALAM.c
#include <stdio.h>

int main(){
int a, b, c, k, l, p;

printf("Nilai A = ");
scanf("%d", &a);
printf("Nilai B = ");
scanf("%d", &b);

p = (b * b) - (a * a);
c = sqrt(p);
k = a + b + c;
l = 0.5 * c * a;
printf("Alas      = %d cm\n", c);
printf("Tinggi    = %d cm\n", a);
printf("Keliling  = %d cm\n", k);
printf("Luas      = %d cm^2\n", l);
return 0;}

File: Python/PRAK205-2210817110005-MIZZUDDINABDISSALAM.py
import math

a = int(input("Nilai A = "))
b = int(input("Nilai B = "))
p = (b * b) - (a * a)
c = math.sqrt(p)
k = a + b + c
l = 0.5 * c * a
print("Alas     =", round(c), "cm")
print("Tinggi   =", a,"cm")
print("Keliling =", round(k), "cm")
print("Luas     =", round(l), "cm^2")